sabnzbd/newsunpack.py

```diff
@@ -1053,6 +1053,10 @@ def seven_extract_core(sevenset, extensions, extraction_path, one_folder, delete
         # We can output a more general error
         ret = 1
         msg = T('ERROR: CRC failed in "%s"') % setname_from_path(sevenset)
+    if ret == 2 and (("Disk full." in output) or ("No space left on device" in output)):
```

Comment:
Let's make this a nice `if / elif`, so first check for the disk-full and then `elif` check for the CRC-failure.